source/cloud-security/amazon/services/supported-services/custom-buckets.rst

```diff
@@ -0,0 +1,135 @@
+
+    #. Create an AWS SQS Queue.
+    #. Configure an S3 bucket to send notifications to the queue for every object creation event.
+
+AWS configuration
+-----------------
+
+Amazon Simple Queue Service
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Set up a **Standard** type SQS Queue with the default configurations and apply the following **Access Policy**:
+
+.. code-block:: json
+
+  {
+  "Version": "2012-10-17",
+  "Id": "example-ID",
+  "Statement": [
+    {
+      "Sid": "example-access-policy",
+      "Effect": "Allow",
+      "Principal": {
+        "Service": "s3.amazonaws.com"
+      },
+      "Action": "SQS:SendMessage",
+      "Resource": "arn:aws:sqs:<region>:<account-id>:<s3-bucket>",
+      "Condition": {
+        "StringEquals": {
+          "aws:SourceAccount": "<account-id>"
+        },
+        "ArnLike": {
+          "aws:SourceArn": "arn:aws:s3:*:*:<s3-bucket>"
+        }
+      }
+    }
+  ]
+  }
+
+  
+.. note::
+     This is an example Access Policy. It can be modified to accept S3 notifications from different ``<account-id>`` values or to apply different conditions. More information `here <https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-overview-of-managing-access.html>`_. 
+
+Amazon S3 and Event Notifications
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The steps to have an S3 bucket reporting creation events are:
+
+#. Configure an S3 bucket as defined in the :ref:`Configuring an S3 Bucket <s3_bucket>` section with the name provided in the previous section.
+#. Once created, go to **Event notifications** and select **Create event notification**. 
+#. In **Event Types**, select **All object create events**. This will generate notifications for any type of event that results in the creation of an object in the bucket.
+#. In the **Destination** section, select **SQS queue** and **Choose from your SQS queues**, choosing the created queue in the previous step.
+
+
+Wazuh Configuration
+-------------------
+
+Buckets Subscriber section in ossec.conf 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Set the configuration inside the section ``<subscriber type="buckets">``. You can find this tag inside the ``<wodle name="aws-s3">`` section of the ``/var/ossec/etc/ossec.conf`` file.
+
+.. code-block:: xml
+
+        <wodle name="aws-s3">
+            <disabled>no</disabled>
+            <interval>1h</interval>
+            <run_on_start>yes</run_on_start>
+            <subscriber type="buckets">
+                <sqs_name>sqs-queue</sqs_name>
+                <aws_profile>profile</aws_profile>
+                <iam_role_arn>arn:aws:iam::example</iam_role_arn>
+                <iam_role_duration>1300</iam_role_duration> 
+                <discard_regex field="field">REGEX</discard_regex>
+                <sts_endpoint>sts-endpoint-IAM</sts_endpoint>
+                <service_endpoint>s3.region.amazonaws.com</service_endpoint>
+            </subscriber>
```

Comment:
In the example configuration, I would include only those configurations that are mandatory so it is easier to understand for users. Similar to the Cloudtrail example configuration:
```XML
<wodle name="aws-s3">
  <disabled>no</disabled>
  <interval>10m</interval>
  <run_on_start>yes</run_on_start>
  <skip_on_error>yes</skip_on_error>
  <bucket type="cloudtrail">
    <name>wazuh-cloudtrail</name>
    <aws_profile>default</aws_profile>
  </bucket>
</wodle>
```

Then a note like this one can be included so they add any additional configuration once the basic use case is working correctly:
![image](https://github.com/wazuh/wazuh-documentation/assets/23361101/6a658d7d-9c8b-4016-bb4e-d641456e7301)